engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/ema/BigNumberEMAOperator.java

```diff
@@ -51,6 +51,7 @@ public boolean isValueValid(long atKey) {
         public void reset() {
             curVal = null;
             lastStamp = NULL_LONG;
+            lastDt = NULL_LONG;
```

Comment:
Might as well do `alpha` and `oneMinusAlpha`, too. You'll never re-use them after a `reset`, and it's better to let GC get rid of them.